### DISCUSSION TITLE
Support to rotate on x and/or y axis only

### DISCUSSION THREAD

**Discussion by sauldom102** [stefanwittwer/remotion-animated#3] [Ideas]
The same way as we can do with `Scale` it would be nice being able to do it with `Rotate` so that we can choose the axis to modify

**Comment by stefanwittwer** [comment]
Good news – control over indiviudal axes in the `Rotate` animation has landed with the support for 3D transformations in Remotion Animated 2.2.0 [(changelog)](https://github.com/stefanwittwer/remotion-animated/releases/tag/v.2.2.0).
